doc/user/content/get-started/install-materialize.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Install Materialize locally"
+description: "Download the Docker image and run Materialize locally for evaluation"
+menu:
+  main:
+    parent: "get-started"
+    weight: 16
+    name: "Install Materialize locally"
+
+---
+
+For testing/evaluation purposes, you can download the Materialize Docker image
+and run Materilize locally in a Docker container.  The Docker image:
+
+- Includes the [Quickstart](/get-started/quickstart) data source
+
+- Supports connecting your data sources
+
+- Provides [community support via
+  Slack](https://materializecommunity.slack.com/join/shared_invite/zt-2bad5ce4i-ZsiPWI5jd7Q9pRDGYj3dkw#/shared-invite/email)
+
+To more fully evaluate
+Materialize and its features, Materialize offers a [free trial
+account](https://materialize.com/register/?utm_campaign=General&utm_source=documentation).
+
+{{< warning >}}
+
+The Docker image is for <redb> testing/evaluation purposes only. It is not
+suitable for production deployments</redb>.  See also [Limitations](#limitations).
+
+{{</ warning >}}
+
+### License and Privacy Policy
+
+- Use of the Docker image is subject to Materialize's [BSL
+  License](https://github.com/MaterializeInc/materialize/blob/main/LICENSE).
+
+- By downloading the Docker image, you are agreeing to Materialize's [privacy
+  policy](https://materialize.com/privacy-policy/).
+
+### Prerequisites
+
+- Docker. If [Docker](https://www.docker.com/) is not installed, refer to its
+[official documentation](https://docs.docker.com/get-docker/) to install.
+
+### Run Materialize in a Docker container
+
+{{< note >}}
+
+- Use of the Docker image is subject to Materialize's [BSL
+   License](https://github.com/MaterializeInc/materialize/blob/main/LICENSE).
+
+- By downloading the Docker image, you are agreeing to Materialize's [privacy
+   policy](https://materialize.com/privacy-policy/).
+
+{{</ note >}}
```

Comment:
Do we need to include this also as a separate section above, if we're already including it here?

Reply:
I figured for those people who will skip that above section, having it repeated right where they will be performing the steps could catch some people.